### PR TITLE
Teams Fx Toolkit Migrated Samples

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/bot-sequential-flow-adaptive-cards/nodejs/.env [hunk 1]
```diff
@@ -1,2 +1,4 @@
 MicrosoftAppId=
-MicrosoftAppPassword=
\ No newline at end of file
+MicrosoftAppPassword=
+MicrosoftAppType=
```
Comment (Prithvi-MSFT) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/.env:3]: Remove extra app settings and update the same in teamslocal yml file

File: samples/bot-sequential-flow-adaptive-cards/nodejs/server/index.js [hunk 2]
```diff
@@ -4,14 +4,15 @@ const cors = require('cors');
 const ENV_FILE = path.join(__dirname, '../.env');
 require('dotenv').config({ path: ENV_FILE });
 
-const PORT = process.env.PORT || 3978;
+//const PORT = process.env.PORT || 3978;
```
Comment (Prithvi-MSFT) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/server/index.js:7]: remove commented code if not in use

File: samples/meetings-app-icon-badging/nodejs/.env [hunk 3]
```diff
@@ -1,3 +1,6 @@
-MicrosoftAppId="<<MicrosoftAppId>>"
-MicrosoftAppPassword="<<MicrosoftAppPassword>>"
-BaseUrl="<<BaseUrl>>"
+MicrosoftAppId=
+MicrosoftAppPassword=
+BaseUrl=
+MicrosoftAppType=
```
Comment (Prithvi-MSFT) [inline samples/meetings-app-icon-badging/nodejs/.env:4]: Remove extra app settings

**Review by Prithvi-MSFT — COMMENTED** [review]
Provided review comments

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine approving

File: samples/bot-formatting-cards/nodejs/teamsapp.yml [hunk 4]
```diff
@@ -3,4 +3,4 @@
 # Visit https://aka.ms/teamsfx-actions for details on actions
 version: 1.0.0
 
-environmentFolderPath: ./env
+environmentFolderPath: ./env
```
Comment (formulahendry) [inline samples/bot-formatting-cards/nodejs/teamsapp.yml:6]: bot-formatting-cards only has this change?

File: samples/meeting-tabs/nodejs/.localConfigs [hunk 5]
```diff
@@ -0,0 +1,5 @@
+BROWSER=none
```
Comment (formulahendry) [inline samples/meeting-tabs/nodejs/.localConfigs:1]: not commit this file

File: samples/meetings-app-icon-badging/nodejs/appPackage/build/manifest.dev.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
+{
```
Comment (formulahendry) [inline samples/meetings-app-icon-badging/nodejs/appPackage/build/manifest.dev.json:1]: not commit the `build` folder

Comment (Mohammed-MSFT) [reply samples/meetings-app-icon-badging/nodejs/appPackage/build/manifest.dev.json:1]: fixed

File: samples/meeting-tabs/nodejs/.localConfigs (hunk 5 above)
Comment (Mohammed-MSFT) [reply samples/meeting-tabs/nodejs/.localConfigs:1]: fixed

File: samples/bot-formatting-cards/nodejs/teamsapp.yml (hunk 4 above)
Comment (Mohammed-MSFT) [reply samples/bot-formatting-cards/nodejs/teamsapp.yml:6]: Files we pushed already
image

Comment (Mohammed-MSFT) [reply samples/bot-formatting-cards/nodejs/teamsapp.yml:6]: fixed

**Review by formulahendry — APPROVED** [review]

**Comment by Mohammed-MSFT** [comment]
Executed Build PipeLine : https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5654136544

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**